api/src/main/java/uk/gov/dhsc/htbhf/claimant/communications/UpcomingBirthdayEmailHandler.java

```diff
@@ -77,25 +76,12 @@ private EmailMessagePayload buildChildTurnsAgeNotificationEmailPayload(PaymentCy
     }
 
     private Map<String, Object> createEmailPersonalisationMapForNextCycle(PaymentCycle paymentCycle, PaymentCycleVoucherEntitlement voucherEntitlement) {
-        Claimant claimant = paymentCycle.getClaim().getClaimant();
-        Map<String, Object> emailPersonalisation = new HashMap<>();
-        emailPersonalisation.put(FIRST_NAME.getTemplateKeyName(), claimant.getFirstName());
-        emailPersonalisation.put(LAST_NAME.getTemplateKeyName(), claimant.getLastName());
-        emailPersonalisation.put(PAYMENT_AMOUNT.getTemplateKeyName(), convertPenceToPounds(voucherEntitlement.getTotalVoucherValueInPence()));
-        emailPersonalisation.put(PREGNANCY_PAYMENT.getTemplateKeyName(), buildPregnancyPaymentAmountSummaryForNextCycle(voucherEntitlement));
+        Map<String, Object> emailPersonalisation = emailMessagePayloadFactory.createCommonEmailPersonalisationMap(paymentCycle, voucherEntitlement);
```

Comment:
Can't you use `createPaymentEmailPersonalisationMap` here?

Reply:
now using createCommonEmailPersonalisationMap

Reply:
My point was that if you use `createPaymentEmailPersonalisationMap` instead of `createCommonEmailPersonalisationMap` then it looks like you can get rid of the two lines below too (children under 1 payment and children under 4 payment).

Reply:
Sadly not - those two lines do something different to the equivalent lines in `createPaymentEmailPersonalisationMap`. We did start out trying to re-use that method.